Library/Formula/cppcms.rb

```diff
@@ -0,0 +1,85 @@
+class Cppcms < Formula
+  desc "A Free High Performance Web Development Framework"
+  homepage "http://cppcms.com/wikipp/en/page/main"
+  url "https://downloads.sourceforge.net/project/cppcms/cppcms/1.0.5/cppcms-1.0.5.tar.bz2"
+  sha256 "84b685977bca97c3e997497f227bd5906adb80555066d811a7046b01c2f51865"
+
+  depends_on "cmake" => :build
+  depends_on "pcre"
+  depends_on "openssl"
+  depends_on :macos => :mavericks
```

Comment:
In a previous PR #41848 @xu-cheng commented:
> ENV.cxx11 in the test won't work as the CFLAGS are never used,  Try to compile the test file in this way:
> ```
> system ENV.cc, "-std=c++11", "-stdlib=libc++", "-lc++", "-o", "test", "test.cpp"
> ```

It works well in `cereal`. However, in this formula I also added these params, but it doesn't work.